src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
             if len(ulog) > 0:
-                if all(key in ulog[-1] for key in ("TriggerEventTypeNumber", "QDate", "JobStartDate",
-                                                   "EnteredCurrentStatus", "MATCH_EXP_JOBGLIDEIN_CMSSite",
-                                                   "WMAgent_JobID")):
+                if all(key in ulog[-1] for key in ("TriggerEventTypeNumber", "QDate", "EnteredCurrentStatus",
```

Comment:
Can you drop the `all` here?  Particularly, `MATCH_EXP_JOBGLIDEIN_CMSSite` is not guaranteed to exist.